src/applications/edu-benefits/complaint-tool/reducers/schoolSelect.js

```diff
@@ -0,0 +1,110 @@
+import {
+  LOAD_SCHOOLS_STARTED,
+  LOAD_SCHOOLS_SUCCEEDED,
+  // LOAD_SCHOOLS_FAILED,
+  SEARCH_INPUT_CHANGE,
+  SELECT_INSTITUTION
+} from '../actions/schoolSearch';
+
+const initialState = {
+  currentPageNumber: 1,
+  institutions: [],
+  institutionQuery: '',
+  institutionSelected: {},
+  pagesCount: 0,
+  searchInputValue: '',
+  searchResultsCount: 0,
+  showInstitutions: false,
+  showInstitutionsLoading: false,
+  showPagination: false,
+  showPaginationLoading: false
+};
+
+export default function schoolSearch(state = initialState, action) {
```

Comment:
Added a reducer so the search state can be restored without trying to push it into the form system. The review page seems to handle the custom component and manual entry predictably. It will require some styling tweaks but I want to wait until the design is finalized before fixing the review page. 
![screen shot 2018-07-26 at 21 56 46](https://user-images.githubusercontent.com/4998130/43300869-d23d7432-911e-11e8-864b-4cfe516dd173.png)